### DISCUSSION TITLE
outline container exiting

### DISCUSSION THREAD

**Discussion by stijn1989** [outline/outline#4408] [Self Hosting] [answered]
I'm trying to run this wiki on docker but it keeps exiting with the following log message:
![image](https://user-images.githubusercontent.com/6877485/201206429-c7c1a348-ac72-4051-a252-b95a26cc35c1.png)

Here is my docker-compose:

https://gist.github.com/stijn1989/552516494a1e6abcaf07944c72297c9e

My env vars looks like this:

https://gist.github.com/stijn1989/2275ac1d7e1de725026ce53286d0581c

The passwords don't contain any special chars. The names for postgres and redis are correct. What am I missing?

**Comment by tommoor** [comment, marked as answer]
This is currently the behavior when no `DATABASE_URL` at all is provided, I'm guessing your env file isn't being loaded by docker-compose for some reason.

**Reply by stijn1989** [reply]
I also forgot the migration part.
